package.json

```diff
@@ -26,9 +26,15 @@
   },
   "scripts": {
     "generate-schema": "generate-airtable-schema && pretty-quick",
-    "start": "babel-node index.js"
+    "start": "babel-node index.js",
+    "test": "mocha --require @babel/register --require babel-polyfill --recursive --file test/billGenerationTests.test.js"
```

Comment:
Added test script. Babel/register and babel/polyfill are necessary for compiling ES6. The recursive tag will be needed when we ad another test file. For now, I am only running the one test file.